source/Module/STIG/Functions.Checklist.ps1

```diff
@@ -78,25 +122,33 @@ function New-StigCheckList
     }
 
     # Values for some of these fields can be read from the .mof file or the DSC results file
-    if ($PSCmdlet.ParameterSetName -eq 'mof')
+    if ($PSCmdlet.ParameterSetName -eq 'single-mof' -or $PSCmdlet.ParameterSetName -eq 'multi-mof')
     {
-        if (-not (Test-Path -Path $ReferenceConfiguration))
+        if (-not (Test-Path -Path $mofFile))
         {
-            throw "$($ReferenceConfiguration) is not a valid path to a configuration (.mof) file. Please provide a valid entry."
+            throw "$($mofFile) is not a valid path to a configuration (.mof) file. Please provide a valid entry."
         }
 
-        $MofString = Get-Content -Path $ReferenceConfiguration -Raw
+        $MofString = Get-Content -Path $mofFile -Raw
         $TargetNode = Get-TargetNodeFromMof($MofString)
```

Comment:
Let's use the PowerShell syntax $targetNode = Get-TargetNodeFromMof -MofString $mofString

Here and on line 154

Reply:
Done.